app/components/content/talk_to_us_component.html.erb

```diff
@@ -7,7 +7,7 @@
       <div>
         <h3 class="heading-l">Call us</h3>
         <p>Our phone line gives you a quick and easy way to get one-off support.</p>
-        <p><a href="tel:08003892500" class="telephone">0800 389 2500</a></p>
+        <p><a href="tel:08003892500" class="telephone">Call us on 0800 389 2500</a></p>
```

Comment:
@philspray What's your view on this change? It feels quite repetitive under the `Call us` h3 - I wonder if we are OK to not make this change here since the page is called 'Get help and support' and we have an h2/h3 as well that indicate what this link is?

Reply:
I agree, I think it's too repetitive with anything else. I've tweaked it.